docs/changes/2629.feature.rst

```diff
@@ -0,0 +1,15 @@
+This commit adds two new hillas-parameter variables:
```

Comment:
The changelog describes a release, not a commit.

Just write something like:

Add two new fields in the Hillas parameters computations: